mindsdb/api/mongo/classes/query_sql.py

```diff
@@ -1,27 +1,40 @@
-from mindsdb.api.mysql.mysql_proxy.controllers.session_controller import SessionController
-from mindsdb.api.mysql.mysql_proxy.executor.executor_commands import ExecuteCommands
+from mindsdb.api.mysql.mysql_proxy.controllers import SessionController
+from mindsdb.api.mysql.mysql_proxy.executor import Executor
+
+
+class SqlServerStub:
+    """This class is just an emulation of Server object,
+    used by Executor.
+    In 'monolithic' mode of MindsDB work the Executor takes
+    some information from the sql server which. Here we emulate
+    this object."""
+
+    def __init__(self, **kwargs):
+        for arg in kwargs:
+            setattr(self, arg, kwargs[arg])
 
 
 def run_sql_command(mindsdb_env, ast_query):
     sql_session = SessionController()
     sql_session.database = 'mindsdb'
+    sqlserver = SqlServerStub(connection_id=-1)
```

Comment:
sql server in mongo server. not good

Reply:
Let me rephrase: sqlserver in Executor is not a good stuff at all, right?